packages/preset-create-react-app/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'pnp-webpack-plugin';
```

Comment:
Can we put this in an `@types` folder instead, named the same as the missing types?